internal/schema/schema_test.go

```diff
@@ -1007,3 +1007,10 @@ func TestInterfaceImplementsInterface(t *testing.T) {
 		})
 	}
 }
+
+func FuzzParse(f *testing.F){
+	f.Fuzz(func(t *testing.T, schemaString string, useStringDescriptions bool){
+		s := schema.New()
+		schema.Parse(s, schemaString, useStringDescriptions)
```

Comment:
When using string descriptions the syntax changes. So maybe we'll end up with a bunch of syntax errors which are expected. I wonder how beneficial is such a test. As in what are the type of errors that we could catch and prevent.